src/components/BenefitPackageTabPanel.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { Paper, Grid, Button } from '@material-ui/core';
+import { Contributions, formatMessage } from '@openimis/fe-core';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+import {
+  BENEFIT_PACKAGE_TABS_LABEL_CONTRIBUTION_KEY,
+  BENEFIT_PACKAGE_TABS_PANEL_CONTRIBUTION_KEY,
+  BENEFIT_PACKAGE_PAYMENTS_TAB_VALUE,
+} from '../constants';
+
+const styles = (theme) => ({
+  paper: theme.paper.paper,
+  tableTitle: theme.table.title,
+  tabs: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  selectedTab: {
+    borderBottom: '4px solid white',
+  },
+  unselectedTab: {
+    borderBottom: '4px solid transparent',
+  },
+  button: {
+    marginLeft: 'auto',
+    padding: theme.spacing(1),
+    fontSize: '0.875rem',
+    textTransform: 'none',
+  },
+});
+
+function BenefitPackageTabPanel({
+  intl, rights, classes,
+}) {
+  const [activeTab, setActiveTab] = useState(BENEFIT_PACKAGE_PAYMENTS_TAB_VALUE);
+
+  const isSelected = (tab) => tab === activeTab;
+
+  const tabStyle = (tab) => (isSelected(tab) ? classes.selectedTab : classes.unselectedTab);
+
+  const handleChange = (_, tab) => setActiveTab(tab);
+
+  return (
+    <Paper className={classes.paper}>
+      <Grid container className={`${classes.tableTitle} ${classes.tabs}`}>
+        <Contributions
+          contributionKey={BENEFIT_PACKAGE_TABS_LABEL_CONTRIBUTION_KEY}
+          intl={intl}
+          rights={rights}
+          value={activeTab}
+          onChange={handleChange}
+          isSelected={isSelected}
+          tabStyle={tabStyle}
+        />
+        <Button variant="contained" color="primary" className={classes.button}>
```

Comment:
Here I see you use 'old approach' with button for custom filtering that was previously ahead of Search criteria 
Currently this 'AdvancedFilter' button should be next to the 'Search Criteria' on right side, please see mocks. I think that when my PRs are merged this should be changed for sure. 
Anyway for that moment is not a huge blocker. Just to remember about design.

Reply:
Yes, you're right. For now I decided to completely remove this 'AdvancedFilter' button. We will add it once your solution will be merged and available.